assets/js/services/format-trade.service.js

```diff
@@ -133,11 +133,16 @@ function formatTrade ($rootScope, $filter, Wallet, MyWallet, currency, Env) {
     let account = accounts && accounts[0];
     let tx = addTradeDetails(trade, account);
 
+    let time;
+    if (/\bINR\b/.test(tx.TOTAL_COST)) time = 'hours';
```

Comment:
@plondon put a check for INR here and pass in the time value via the formatted trade. this leaves `trade-summary` unchanged and partner agnostic.